### PR TITLE
\hyperlink{stop}{STOP}

### PR DESCRIPTION
**Pull request by jamesray1**

### CODE REVIEW

File: Paper.tex [hunk 1]
```diff
@@ -1016,7 +1016,7 @@ \subsubsection{Normal Halting}\hypertarget{normal_halting_function_H}{}\linkdest
 \begin{equation}
 H(\boldsymbol{\mu}, I) \equiv \begin{cases}
 H_{\text{\tiny RETURN}}(\boldsymbol{\mu}) & \text{if} \quad w \in \{\text{\small \hyperlink{RETURN}{RETURN}}, \text{\small REVERT}\} \\
-() & \text{if} \quad w \in \{ \text{\small STOP}, \text{\small \hyperlink{selfdestruct}{SELFDESTRUCT}} \} \\
+() & \text{if} \quad w \in \{ \text{\small text{\hyperlink{stop}{STOP}}, \text{\small \hyperlink{selfdestruct}{SELFDESTRUCT}} \} \\
```
Comment (pirapira) [inline Paper.tex]: Why `text` not `\text`?

Comment (jamesray1) [reply Paper.tex]: Fixed.

**Comment by pirapira** [comment]
https://travis-ci.org/ethereum/yellowpaper/builds/342666077#L1545

```
Runaway argument?
{\small \text {\hyperlink {stop}{STOP}}, \text {\small \hyperlink {se\ETC.
! Paragraph ended before \text@ was complete.
<to be read again> 
                   \par 
l.1022 
       
!  ==> Fatal error occurred, no output PDF file produced!
Transcript written on build/Paper.log.
```

**Comment by jamesray1** [comment]
Fixed.

**Comment by pirapira** [comment]
```
Runaway argument?
{\small {\hyperlink {stop}{STOP}}, \text {\small \hyperlink {selfdest\ETC.
! Paragraph ended before \text@ was complete.
<to be read again> 
                   \par 
l.1022 
       
!  ==> Fatal error occurred, no output PDF file produced!
Transcript written on build/Paper.log.
```

**Pull request merged**